demo/components/ResizableSection.vue

```diff
@@ -21,32 +21,46 @@
 </template>
 
 <script lang="ts" setup>
-  import { toPixels } from '@prefecthq/prefect-design'
-  import { computed, ref } from 'vue'
+  import { useColorTheme, toPixels } from '@prefecthq/prefect-design'
+  import { computed, ref, nextTick } from 'vue'
   import ResizeIcon from '@/demo/components/ResizeIcon.svg'
+  import { activeColorMode } from '@/demo/utilities/colorMode'
+  import { getColorModeClass } from '@/utilities'
 
-  const container = ref<HTMLDivElement>()
-  const dragging = ref(false)
-  const contentWidth = ref<number>()
   const minWidth = 200
   const handleWidth = 24
   const handleWidthPx = `${handleWidth}px`
 
+  const { value: colorTheme } = useColorTheme()
+
+  const container = ref<HTMLDivElement>()
+  const dragging = ref(false)
+  const contentWidth = ref<number>()
+  const show = ref(true)
+
+
   const classes = computed(() => ({
+    content: {
+      [getColorModeClass(activeColorMode.value)]: true,
+      'dark': colorTheme.value === 'dark',
+      'light': colorTheme.value === 'light',
+      'bg-background-700': colorTheme.value === 'light',
+      'bg-background-400': colorTheme.value === 'dark',
+    },
```

Comment:
```suggestion
    content: [getColorModeClass(activeColorMode.value, {
      'dark': colorTheme.value === 'dark',
      'light': colorTheme.value === 'light',
      'bg-background-700': colorTheme.value === 'light',
      'bg-background-400': colorTheme.value === 'dark',
    }],
```

Reply:
Ah actually `p-frame` is lazily looking for `Record<string, unknown>` as a `bodyClass` type which means we can't use this syntax

Reply:
🤔  these should be the same?  
```
{ 
  [getColorModeClass(activeColorMode.value)]: true
}
```
```
[getColorModeClass(activeColorMode.value]
```

Reply:
oh I get your comment now. Looking at that

Reply:
Yeah it was probably cause I wrote the prop that way tbh but I think there's a good reason to; passing around the vue class and style types is really slow for TS cause they're huge

Reply:
Vue has some helper types for this sort of this. Anyway, carry on. I'll create an issue in prefect-design to support vue style dynamic classes

Reply:
https://github.com/PrefectHQ/prefect-design/issues/603